assets/src/scss/pages/_listing-page.scss

```diff
@@ -120,27 +120,15 @@
 .query-list-item-content p {
   font-size: $font-size-sm;
   margin-bottom: $sp-1;
-  line-height: 25px;
-  -webkit-line-clamp: 3;
-
-  @include medium-and-up {
-    -webkit-line-clamp: 2;
-  }
-
-  @include large-and-up {
-    -webkit-line-clamp: 3;
-  }
+  line-height: 145%;
```

Comment:
Looking at [Figma](https://www.figma.com/file/9NtbY8n3at8uOEJTsLrETb/P4-Design-System?type=design&node-id=3200-4972&mode=dev), I think we can use `--line-height-m--font-family-secondary` instead.